### PR TITLE
feat: add homepage

### PR DESCRIPTION
**Pull request by Sahil25061999**
Added homepage

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+  align-items: flex-start;
+}
+
+.hero-section .btn {
+  box-shadow: none;
+}
+
+.hero-container {
+  width: 50vw;
+  min-width: 350px;
```
Comment (Anurag021) [inline index.css]: it would be better if you can use rem instead of px

**Review by Anurag021 — APPROVED** [review]
Code looks good to me, just on the image part, some images are quite heavy (eg. background image is 12 mb) it will going to affect the performance of the website, it would be better if you can reduce it.

File: index.css (hunk 1 above)
Comment (Sahil25061999) [reply index.css]: Thanks, will change it.

**Review by prankurpandeyy — APPROVED** [review]

**Comment by prankurpandeyy** [comment]
looks good to me , add placeholder in search bar .

File: index.css [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+font-family: 'Montserrat', sans-serif; */
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  font-family: 'Montserrat', sans-serif;
```
Comment (pradipgharge) [inline index.css:18]: You can add font family to body itself, rather than individual elements.

**Review by pradipgharge — COMMENTED** [review]
Rest code looks good to merge !

File: index.css (hunk 2 above)
Comment (Sahil25061999) [reply index.css:18]: i wanted the fonts to apply to heading elements only 😊

**Pull request merged**